app.json

```diff
@@ -0,0 +1,10 @@
+{
+  "addons": [
+    "logentries:entry_plus"
+  ],
+  "buildpacks": [
+    {
+      "url": "https://github.com/kr/heroku-buildpack-go.git"
```

Comment:
Heroku [now officially support](http://blog.heroku.com/archives/2015/7/7/go_support_now_official_on_heroku) Go and have [their own buildpack](https://github.com/heroku/heroku-buildpack-go). I think we can remove this and let Heroku automatically detect the language.